code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -37,6 +48,8 @@
 	can_collar = TRUE
 	gold_core_spawnable = FRIENDLY_SPAWN
 	var/chew_probability = 1
+	/// Is our mouse smart?
+	var/sentient = FALSE
```

Comment:
Where is this getting set? Is this designed for adminbus?

Reply:
In sentience event and potion from xeno